_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+- Java
+
+#### Print support
+
+ApiaryUI is fully print friendly. You can easily print the rendered documentation directly on paper or into PDF.
+
+
+#### Accessibility
+
+ApiaryUI complies with latest accessibility standards. We care about correctness of HTML semantics and proper use of Aria attributes.
+ApiaryUI is compatible with screen readers and even specially-abled people should navigate in it with no problems.
+Full support for touch events is guaranteed.
+
+##### Keyboard navigation
+
+ApiaryUI supports keyboard navigation for people who cannot use a mouse or other pointing device, 
+or who want to use a keyboard as much as possible.
+
+
+#### Mobile First (building for the future)
+
+Mobile devices are already eclipsing desktops in adoption and internet use — that's why ApiaryUI was built from
+the ground up to support any kind of device, any size screen, with any resolution. 
+ApiaryUI behaves and looks differently on mobile and desktop resolutions to maximize the user experience. 
+
+#### Error Boundaries
+
+ApiaryUI has full support for error containment and reporting. When the ApiaryUI intercept an unexpected failure
+it will contain it and display the Error Boundary view. This view contains button for downloading
+the complete Error Report. 
+
+
+### Next steps
+
+#### Speed
+
+We're currently parsing your Api Description Documentation on every browser view. This makes
+displaying of your API Projects via ApiaryUI unnecessarily slower. We plan to cache the result
+of the parsing and skip the parsing phase of ApiaryUI. This should rapidly increase the
+rendering times.  
+
+
+#### Deprecation strategies
+
+As I mentioned earlier, we are in the process of forming deprecation strategies for 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) 
+and [Embed](https://help.apiary.io/tools/embed/). 
+
+#### Development of missing features
+
+We'll be continuously delivering missing features from [Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) into ApiaryUI.
+To learn more about the missing features, please refer to [ApiaryUI help page](https://help.apiary.io/tools/interactive-documentation-v4/).
+
+
+I would like to wish all Apiary users Mary Christmas and Happy new Year. We are thrilled to 
+welcome all Apiary users to try the new ApiaryUI - Interactive Documentation v4. 
+Follow this [help page](https://help.apiary.io/tools/interactive-documentation-v4/) to get information how to enable ApiaryUI on your existing API Projects.
+The help page also contains some additional information about features from Interactive Documentation v3
+that are currently missing in ApiaryUI.
+
+
+### Attributions
+
+I would like to give special thanks to Patrick Silva, Radek Podrazsky and Standa Opichal.
+These guys formed a core of new Documentation Team, did an exceptional job and make the delivery of ApiaryUI possible.
+
+Thank you guys and keep up the good work!
```

Comment:
While I like and fully support the attribution of the team, ending the article with it makes this a bit self-serving and worth an internal announcement and/or "how v4 was baked" blog post (which I would totally love to read and if you are up for it, I think it's totally worth writing!), but not a product announcement. 

I think the article should end with CTA for trying the documentation out and with thank you to the users since they are the audience of the article.

I also think both thank yous are possible within the same article, but it requires some thought and reshuffling of the content.